day_6/exercises/burrito.rb

```diff
@@ -4,16 +4,16 @@
 # 2. remove_topping
 # 3. change_protein
 
-class Burrito
-  attr_reader :protein, :base, :toppings
-  def initialize(protein, base, toppings)
+class Burrito # => class
```

Comment:
Hey Tyler! For this exercise we'd like to see you attempt to change the values that are initialized on line 16 with methods that exist within the class. Ruby has built in array methods that can handle both deleting and adding items to an array. You can also change the value of the protein with a method by setting the instance variable equal to whatever protein you pass into the method as an argument. You did a similar thing with the `speak` method in your `Classes_and_objects.rb` file. We'd like you to add some methods to this class that you can then call after you initialize the class on line 16. Let me know if you have questions about this!